Barotrauma/BarotraumaShared/SharedSource/StatusEffects/StatusEffect.cs

```diff
@@ -1371,7 +1371,7 @@ public bool ShouldWaitForInterval(Entity entity, float deltaTime)
             return false;
         }
 
-        public virtual void Apply(ActionType type, float deltaTime, Entity entity, ISerializableEntity target, Vector2? worldPosition = null)
+        public virtual void Apply(ActionType type, float deltaTime, Entity entity, ISerializableEntity target, Vector2? worldPosition = null, character = null)
```

Comment:
Not needed here, but adding it has some consistency with the actual method that will use it. Not a strong opinion though